### DISCUSSION TITLE
How to send g.711a from webrtcclientmaster

### DISCUSSION THREAD

**Discussion by PranavSanthosh186** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1922] [Q&A] [answered]
Instead of opus i need to send g.711a format audio files how can i do it?
Pls mention the steps

**Comment by disa6302** [comment, marked as answer]
Hello,

You need to set the relevant transceiver here: https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/v1.9.1/samples/Common.c#L539-L546. The codec enums can be found here: 
https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/v1.9.1/src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h#L789-L796
